### PR TITLE
Introduce chezmoi

### PR DESCRIPTION
**Pull request by johnmanjiro13**
https://www.chezmoi.io/

### CODE REVIEW

File: dot_config/chezmoi/chezmoi.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[edit]
```
Comment (johnmanjiro13) [inline dot_config/chezmoi/chezmoi.toml]: It needs a template file.

Comment (twpayne) [reply dot_config/chezmoi/chezmoi.toml]: Running

```console
$ chezmoi cd
$ mv dot_config/chezmoi/chezmoi.toml .chezmoi.toml.tmpl
$ rm -rf dot_config/chezmoi
```

will achieve this.

Comment (johnmanjiro13) [reply dot_config/chezmoi/chezmoi.toml]: Thank you!

Comment (johnmanjiro13) [reply dot_config/chezmoi/chezmoi.toml]: `chezmoi doctor` says all files are ok.
```console
ok        suspicious-entries   no suspicious entries
```

**Pull request merged**